code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -1,159 +1,147 @@
-
-/obj/item/device/encryptionkey/
+/obj/item/device/encryptionkey
 	name = "standard encryption key"
 	desc = "An encryption key for a radio headset. Contains cypherkeys."
 	icon = 'icons/obj/radio.dmi'
 	icon_state = "cypherkey"
 	item_state = ""
-	w_class = 1
+	w_class = ITEMSIZE_TINY
 	slot_flags = SLOT_EARS
-	var/translate_binary = 0
-	var/translate_hive = 0
-	var/translate_hivenet = 0
-	var/syndie = 0
-	var/list/channels = list()
+	var/translate_binary = FALSE
+	var/translate_hive = FALSE
+	var/translate_hivenet = FALSE
+	var/syndie = FALSE // Signifies that it de-crypts Syndicate transmissions
+	var/list/channels = list("Common" = TRUE, "Entertainment" = TRUE)
```

Comment:
Entertainment should start off imho.

Reply:
i think it's best to leave it on, so whoever plans to use it can just begin talking. anyone who isn't interested can turn it off